src/test/java/org/opensearch/geospatial/ip2geo/Ip2GeoTestCase.java

```diff
@@ -161,6 +161,11 @@ public File sampleIp2GeoFile() {
         return new File(this.getClass().getClassLoader().getResource("ip2geo/sample_valid.csv").getFile());
     }
 
+    public long randomPositiveLong() {
```

Comment:
can we make this function protected?

Reply:
also why all the functions in this class are public? Can we make this protected given that it is an abstract test class.